client/src/mocks/mockUseAuthProvider.tsx

```diff
@@ -1,19 +1,20 @@
-import { FunctionComponent } from 'react';
-import { AuthContext } from '../context/useAuthContext';
-import { mockLoggedInUser } from './mockUser';
+// import { FunctionComponent } from 'react';
```

Comment:
did we want to remove these?

Reply:
This one also is commented out because of above reason. But I will change the types of User interface, so we don't get unnecessary ts errors.